ramani-maplibre/src/main/java/org/ramani/compose/Symbol.kt

```diff
@@ -31,7 +31,7 @@ fun Symbol(
     color: String,
     isDraggable: Boolean,
     zIndex: Int = 0,
-    imageId: Int? = null,
+    imageId: Int? = R.drawable.maplibre_marker_icon_default,
```

Comment:
I think that it makes sense to default to some icon instead of having the symbol be invisible.